crates/rover-client/src/operations/graph/introspect/schema.rs

```diff
@@ -1405,24 +1405,28 @@ mod tests {
           "How much the product weighs in kg"
           weight: Int @deprecated(reason: "Not all product's have a weight")
           "A simple list of all reviews for a product"
-          reviews: [Review] @deprecated(reason: "The `reviews` field on product is deprecated to roll over the return
-        type from a simple list to a paginated list. The easiest way to fix your
-        operations is to alias the new field `reviewList` to `review`:
-
-          {
-            ... on Product {
-              reviews: reviewList {
-                edges {
-                  review {
-                    body
+          reviews: [Review] @deprecated(reason:
+          """
+          The `reviews` field on product is deprecated to roll over the return
+          type from a simple list to a paginated list. The easiest way to fix your
+          operations is to alias the new field `reviewList` to `review`:
+          
+            {
+              ... on Product {
+                reviews: reviewList {
+                  edges {
+                    review {
+                      body
+                    }
```

Comment:
I've made changes to `apollo-encoder` to use either StringValue or BlockStringValue for arguments as well. This is useful for when a reason string contains `"this is a "reason""`, as it does not parse in graphql-js. It will, however parse this:
```
"""
this is a "reason"
"""
```